nav2_behavior_tree/include/nav2_behavior_tree/plugins/action/clear_costmap_service.hpp

```diff
@@ -49,6 +50,19 @@ class ClearEntireCostmapService : public BtServiceNode<nav2_msgs::srv::ClearEnti
    * @return BT::NodeStatus Status of tick execution
    */
   void on_tick() override;
+
+  /**
+   * @brief Creates list of BT ports
+   * @return BT::PortsList Containing basic ports along with node-specific ports
+   */
+  static BT::PortsList providedPorts()
+  {
+    return providedBasicPorts(
+      {
+        BT::InputPort<std::vector<std::string>>("plugins",
```

Comment:
Please also update the corresponding entries in the [nav2_tree_nodes.xml](https://github.com/ros-navigation/navigation2/blob/16e63a46c0e25339dfddee468589b456cf0af7df/nav2_behavior_tree/nav2_tree_nodes.xml#L67) to declare a new `plugins` input port for all the four BtServiceNode's